src/lib/bootloader/grub2_widgets.rb

```diff
@@ -535,9 +522,9 @@ def store
         console_value = Yast::UI.QueryWidget(Id(:console_args), :Value)
         BootloaderFactory.current.enable_serial_console(console_value)
       elsif use_gfxterm
-        grub_default.terminal = :gfxterm
+        grub_default.terminal = [:gfxterm]
```

Comment:
this is in fact wrong as currently the code does not allow to set both to true ( https://github.com/yast/yast-bootloader/pull/528/files#diff-8a7a995c9c06bae0fbc6acd4ccd71a23R519 ). This can be now removed and it should here add/remove `:gfxterm` and `:console` according to user configuration.

Reply:
ah, I mention gfxterm and console, but mean gfxterm and serial console which can be used in parallel. For gfxterm and console it does not make much sense.